.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -29,15 +29,15 @@ steps:
       parameters:
         packageId: ${{parameters.desktopId}}
         packageVersion: ${{parameters.npmVersion}}
-        slices: ''
+        slices: ${{ coalesce(parameters.slices, '("x64.Debug", "x86.Debug", "ARM64.Debug", "x64.Release","x86.Release", "ARM64.Release")') }}
```

Comment:
What does this achieve?

Does it add append `parameters.slices` with the value to its right?

Reply:
It will prefer parameters.slice if not an empty string, otherwise pick the values on the right.

This is sort of a replacement of the previous default parameter, since we want different default slices for desktop vs MSRN. I.e. the MSRN NuGet package doesn't contain debug bits, but Office does.